pkg/beacon/entry/submission.go

```diff
@@ -46,45 +48,27 @@ func (res *relayEntrySubmitter) submitRelayEntry(
 	for {
 		select {
 		case blockNumber := <-eligibleToSubmitWaiter:
-			// Member becomes eligible to submit the result.
-			errorChannel := make(chan error)
-			defer close(errorChannel)
-
 			logger.Infof(
-				"[member:%v] submitting relay entry [0x%x] on behalf of group "+
-					"[0x%x] at block [%v]",
+				"[member:%v] firing relay entry [0x%x] submission "+
+					"on behalf of group [0x%x] at block [%v]",
```

Comment:
I would leave the previous version to be honest. They are *submitting*, it does not mean the submission will succeed.

Reply:
See 364b53a7